javascript-timers/interval.js

```diff
@@ -0,0 +1,11 @@
+var h1 = document.querySelector('.countdown-display');
+
+var timer = 4;
+
+function setText() {
+  if (timer < 1) {
+    h1.textContent = '~Earth Beeeelooowww Us~';
+  } else h1.textContent = timer;
+  timer--;
+}
+setInterval(setText, 1000);
```

Comment:
make sure you clear the interval for this one once the functionality is complete